tests/v0_tests/test_add_documents.py

```diff
@@ -504,14 +515,45 @@ def test_multimodal_combination_doc(self):
         # A space in the middle
         if self.IS_MULTI_INSTANCE:
             self.warm_request(self.client.index(self.index_name_1).search,
-            "", filter_string="space\ field.space\ child\ 1:(search this with space)")
+                              "", filter_string="space\ field.space\ child\ 1:(search this with space)")
 
         space_filtering_res = self.client.index(self.index_name_1).search(
             "", filter_string="space\ field.space\ child\ 1:(search this with space)")
         assert space_filtering_res["hits"][0]["_id"] == "111"
 
         if self.IS_MULTI_INSTANCE:
-            self.warm_request(self.client.index(self.index_name_1).search,"")
+            self.warm_request(self.client.index(self.index_name_1).search, "")
 
         space_tensor_res = self.client.index(self.index_name_1).search("")
-        assert space_tensor_res["hits"][0]["_id"] == "111"
\ No newline at end of file
+        assert space_tensor_res["hits"][0]["_id"] == "111"
+
+
+class TestAddDocumentsImageDownloadHeaders(MarqoTestCase):
+    def setUp(self) -> None:
+        self.client = Client(**self.client_settings)
+        self.index_name_1 = "my-test-index-1"
+        try:
+            self.client.delete_index(self.index_name_1)
+        except MarqoApiError as s:
+            pass
+
+    def tearDown(self) -> None:
+        try:
+            self.client.delete_index(self.index_name_1)
+        except MarqoApiError as s:
+            pass
+
+    def test_add_docs_image_download_headers(self):
```

Comment:
I think this test can go under `TestAddDocuments` and doesn't require a new class